src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/PlatformAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+
+    /// <summary>
+    /// Base type for all platform-specific attributes. Primarily used to allow grouping
+    /// in documentation.
+    /// </summary>
+    public abstract class PlatformAttribute : Attribute
```

Comment:
@terrajobst Should this be `OSPlatformAttribute` or is `PlatformAttribute` right?

Reply:
It should be `OSPlatformAttribute` https://github.com/dotnet/runtime/issues/33331#issuecomment-650326500